pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/TimeAverageMessageData.java

```diff
@@ -41,15 +41,20 @@ public class TimeAverageMessageData {
     // The average message rate out per second.
     private double msgRateOut;
 
+    // Consider the throughput equal if difference is less than 100 KB/s
+    private static final double throughputDifferenceThreshold = 1e5;
+    // Consider the msgRate equal if the difference is less than 100
+    private static final double msgRateDifferenceThreshold = 100;
```

Comment:
What's the justification for these vals?

Reply:
I imitate the logic implemented by 
https://github.com/apache/pulsar/blob/d099ac4fa2f217b9c5f0a5e660c83048e829c5d7/pulsar-client-admin-api/src/main/java/org/apache/pulsar/policies/data/loadbalancer/NamespaceBundleStats.java#L40-L47